docs/DeveloperGuide.adoc

```diff
@@ -849,9 +849,16 @@ _{More to be added}_
 [appendix]
 == Non Functional Requirements
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `1.8.0_60` or higher installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  The system should work on Windows, Linux and macOS as long as it has Java `1.8.0_60` or higher installed.
+.  A user (vet or vet assistant) with above average typing speed for regular English text (i.e. not code, not system administrative commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  The system should be accessible, even without any Internet connection (or with poor Internet connection).
```

Comment:
@CS2103JAN2018-F14-B2/developers Can I double check whether our application will only be used locally on the desktop, or we require Internet connection to update the databases? For doctors, I know they have to connect to the Internet because of MOH's requirements to update patient's information. What about vets? Do they need the Internet for collaboration purposes?

Reply:
I just checked the project constraints:

"Constraint-Human-Editable-File: The data should be stored locally and should be in a human editable text file. Use of relational or No-SQL databases are not allowed."

In the User Guide I specified internet connection is required for syncing data to support multi-user (for v2.0 though).

Reply:
@Aquarinte Thanks for the clarification! @CS2103JAN2018-F14-B2/developers Project constraints are found at https://nus-cs2103-ay1718s2.github.io/website/ > "Week 1" > "Todo" > "Admin info to read" (FYI if you were like me and cannot find the constraints because you were looking at addressbook-level4).